src/views/catalog/customize/components/CustomizeSource/ExpandableCustomizeSourceSection.scss

```diff
@@ -0,0 +1,9 @@
+.expandable-customize-source-section__stack-item-storage {
+  .pf-l-flex {
+    --pf-l-flex--AlignItems: center;
```

Comment:
if you change it to baseline instead of center does it look better?

Reply:
I think baseline was default..